pom.xml

```diff
@@ -127,22 +92,15 @@
       </plugin>
 
       <plugin>
-        <groupId>io.syndesis</groupId>
-        <artifactId>syndesis-maven-plugin</artifactId>
-        <version>${syndesis.version}</version>
+        <groupId>io.syndesis.extension</groupId>
+        <artifactId>extension-maven-plugin</artifactId>
```

Comment:
I think we need to specify the version here, as far as I understand, the plugin version is not imported from the BOM.